assets/sass/components/global/_googlesitekit-header.scss

```diff
@@ -51,7 +66,68 @@
 		}
 	}
 
-	.googlesitekit-header__dropdown {
-		margin-right: -8px;
+	.googlesitekit-dropdown-menu {
+		display: inline-block;
+
+		.mdc-button__icon {
+			margin-right: 12px;
+		}
+
+		.mdc-button__icon,
+		.mdc-button__icon--image {
+			height: 28px;
+			width: 28px;
+		}
+
+		.mdc-button__label {
+			font-weight: $fw-primary-normal;
+		}
+	}
+
+	.googlesitekit-date-range-selector {
+		margin-right: 8px;
+
+		@media (max-width: $bp-xsmallOnly) {
+			margin-right: 0;
+		}
+
+		@media (min-width: $bp-wpAdminBarTablet) {
+			margin-right: 24px;
+		}
+
+		svg {
+			margin-right: 8px;
+		}
+
+		.mdc-button--dropdown {
+			padding-right: 25px;
+
+			.mdc-button__label {
+
+				@media (max-width: $bp-xsmallOnly) {
+					display: none;
+				}
+			}
```

Comment:
According to the design this shouldn't be hidden in mobile

![image](https://user-images.githubusercontent.com/1621608/100206392-eef4cb80-2f0e-11eb-8939-b2bd42525818.png)

![image](https://user-images.githubusercontent.com/1621608/100206396-f0be8f00-2f0e-11eb-8a44-3555b356aaf4.png)